survey/4-ai.yml

```diff
@@ -87,3 +87,81 @@ questions:
       - IBM Watson
       - Scikit-Learn
       - Other
+
+  - label: "Which vector databases have you used or are familiar with?"
```

Comment:
A bit skeptical about this one because its a niche question for people who actively ai developers + vector database are not exclusive to AI  we can just move the question to more general section or be more precise about the usage.